plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -99,14 +106,15 @@ public DruidJdbcClient(BaseJdbcConfig config, ConnectionFactory connectionFactor
                 if (tableHandles.isEmpty()) {
                     return Optional.empty();
                 }
+
                 return Optional.of(
```

Comment:
Can this be removed now?

Doesn't look possible now since ambiguous tables and schemas will cause an exception to be thrown from `toRemoteXxx` method.

Take a look at `BaseJdbcClient` for how it can look now.

Reply:
As discussed offline with @dheerajkulakarni looks like `getTables` can return multiple tables (even if their names do not match exactly) because the Avatica driver doesn't handle escaping search strings - so this filtering logic will need to remain.

Please mark this as resolved.

Reply:
Sure @hashhar!